bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/renovator/MovingAsset.java

```diff
@@ -61,6 +67,14 @@ public void move(ReplicatorQueue replicatorQueue, ResourceResolver rr) throws Il
             session.getWorkspace().getObservationManager().setUserData("changedByWorkflowProcess");
             session.move(getSourcePath(), getDestinationPath());
             session.save();
+            if (session.nodeExists(getDestinationPath())) {
+                Node originalAssetJcrContentNode = session
+                        .getNode(getDestinationPath() + "/" + JcrConstants.JCR_CONTENT);
+
+                JcrUtil.setProperty(originalAssetJcrContentNode, JcrConstants.JCR_LASTMODIFIED, new Date());
+                JcrUtil.setProperty(originalAssetJcrContentNode, JcrConstants.JCR_LAST_MODIFIED_BY,
+                        DEFAULT_LAST_MODIFIED_BY);
```

Comment:
I assume there isnt a clean way to get the user who kicked off the Renovation?

Reply:
That's the user that runs renovator so pull it from the resource resolver in hand.

Reply:
@davidjgonzalez / @badvision : The driver behind adding "user name" is to have some constant value updated on an asset which acts as an indicator for our Asset Management workflow setup not to run workflows and push to scene7. This would be a similar concern for any organization who have their DAM integrated with Scene7 or any external asset storage system.

we have no concern retrieving username from resource resolver but the goal we are trying to achieve is put something on an asset which gives an indicator that it was touched by Renovator.

CC : @mxp737